libmattermost.c

```diff
@@ -4206,6 +4237,9 @@ PurpleConnection *pc
 	act = purple_protocol_action_new(_("Search for friends..."), mm_search_users);
 	m = g_list_append(m, act);
 
+	act = purple_protocol_action_new(_("Room List"), mm_roomlist_show);

```

Comment:
Tools->Room List will show selection for all active accounts, while in Account-> Room List the account is preselected and the action executed without need of additional clicks: but if you think this is overdoing ... please remove.